cdc/owner/owner.go

```diff
@@ -794,8 +808,13 @@ func (o *ownerImpl) calculateGCSafepoint(state *orchestrator.GlobalReactorState)
 		if changefeedState.Info == nil {
 			continue
 		}
+
```

Comment:
will the changefeed  be removed from the state.changefeeds if the changefeed failed ?

Reply:
the failed changefeed won't be removed from the state.changfeeds